usagov/vote-gov-drupal
# VOTE-2357 initialize a custom nvrf page route

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2357

## Description

Create a custom page for the nvrf app to live.

## Deployment and testing

### Post-deploy steps

1.  cd into the vote_nvrf module and run `npm i` and `npm run build`
2. `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/register/washington/form and make sure the app loads with Washington in the content
2. visit http://vote-gov.lndo.site/register/guam/form and make sure this results in a 404
3. visit http://vote-gov.lndo.site/ko/register/washington/form and make sure this results in a 404

## Checklist for the Developer




## Checklist for the Peer Reviewers
